unreleased/add-azureblob-receiver.yaml

```diff
@@ -1,16 +0,0 @@
-# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
-change_type: new_component
-
-# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
-component: azureblobreciver
-
-# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
-note: Add a new component `azureblobreciver`
-
-# One or more tracking issues related to the change
-issues: [8834]
-
-# (Optional) One or more lines of additional information to render under the primary note.
-# These lines will be padded with 2 spaces and then inserted directly into the document.
-# Use pipe (|) for multiline entries.
-subtext: Add a new component `azureblobreceiver` that reads logs and traces from Azure Blob Storage
```

Comment:
Are they really duplicates or maybe missed? Anyway, it's probably a good candidate for another PR

Reply:
Sorry, apparently another PR leaked into this one. Thanks for catching this.